cal/__init__.py

```diff
@@ -66,13 +66,7 @@ def home():
     return render_template('index.html', events=events)
 
 
-# TO REMOVE: Manual update route
-@app.route('/update')
-def update():
-    update_fb_events()
-    return jsonify({"success": True})
-
-
+# TO REMOVE
 @app.route('/events')
```

Comment:
We don't actually want to remove this route (although we probably will change our routing structure in the future). It's where React populates its initial event list.